voluncheer/opportunityboard/tests_views.py

```diff
@@ -145,15 +148,26 @@ def test_deregister_volunteer(self):
 class OrganizationViewTestCase(TestCase):
     def setUp(self):
         super().setUp()
+        self.user = get_user_model().objects.create_user(
+            email="test@voluncheer.com",
+            password="secret",
+            type=UserType.VOLUNTEER,
+        )
+        self.user.save()
+        self.client.login(email="test@voluncheer.com", password="secret")
+        self.vol2 = Volunteer.objects.create(
+            user=self.user,
+            first_name="Luke",
+            last_name="Skywalker",
+            date_of_birth="1955-09-25",
+        ).save()
 
     def test_organization_view(self):
```

Comment:
Please add a docstring here to explain the test.

Reply:
Done